cmd/cork/create.go

```diff
@@ -122,7 +124,9 @@ func init() {
 
 	creationFlags = pflag.NewFlagSet("creation", pflag.ExitOnError)
 	creationFlags.StringVar(&sdkUrlPath,
-		"sdk-url-path", "/flatcar-jenkins/sdk", "SDK URL path")
+		"sdk-url-path", "sdk", "SDK URL path")
```

Comment:
Any reason to change this?

Reply:
Oh, I think I know now - the default SDK URL is `https://mirror.release.flatcar-linux.net`, and there we store the SDKs in the `sdk` directory.

Nevermind then. :)

Reply:
Exactly - it decreases user friction and the CI always passes the `--sdk-url-path` value.